docs/200_ContextGuard.md

```diff
@@ -0,0 +1,40 @@
+# Context Guard
+
+The context guard add an additional layer of protection and controll ontop of the set of your indexing rules. It allows you to filter the result and avoid indexing unvanted content.
+
+To  implement a guard add a service that would implement `ContextGuardInterface`.
+
+## Example Setup
+
+``` yaml
+services:
+  AppBundle\DynamicSearch\Guard\DefaultContextGuard:
```

Comment:
Please use `App` Namespace here (Symfony 5 Standard)

Reply:
I've updated the namespace and the path (to `config/config` in v5). Some more changes have been done to proovide a working setup based on the official demo.